proto/worker.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "github.com/dmitsh/gravitest/proto";
+
+service Worker {
+  rpc StartProcess (StartProcessRequest) returns (JobId);
+  rpc GetProcessStatus (JobId) returns (Status);
```

Comment:
Does only process status is helpful from user perspective ? For example if a job terminated with an error obtaining a process exit status code will be useful.